labgrid/driver/lxaiobusdriver.py

```diff
@@ -0,0 +1,62 @@
+import attr
+
+import requests
+
+from ..factory import target_factory
+from ..protocol import DigitalOutputProtocol
+from ..util.proxy import proxymanager
+from .common import Driver
+from .exception import ExecutionError
+from ..step import step
+
+@target_factory.reg_driver
+@attr.s(eq=False)
+class LXAIOBusPIODriver(Driver, DigitalOutputProtocol):
+    bindings = {
+            "pio": {"LXAIOBusPIO", "NetworkLXAIOBusPIO"},
+    }
+
+    def __attrs_post_init__(self):
+        super().__attrs_post_init__()
+        self._host = None
+        self._port = None
+
+    def on_activate(self):
+        # we can only forward if the backend knows which port to use
+        self._host, self._port = proxymanager.get_host_and_port(self.pio)
+
+    def on_deactivate(self):
+        self._host = None
+        self._port = None
+
+    @Driver.check_active
+    @step(args=['status'])
+    def set(self, status):
+        if self.pio.invert:
+            status = not status
+        r = requests.post(
+                'http://{}:{}/nodes/{}/pins/{}/'.format(self._host, self._port, self.pio.node, self.pio.pin),
+                data={'value': '1' if status else '0'}
+        )
+        r.raise_for_status()
+        j = r.json()
+        if j["code"] != 0:
+            raise ExecutionError("failed to set value: {}".format(j["error_message"]))
+
+    @Driver.check_active
+    @step(result=['True'])
+    def get(self):
+        r = requests.get(
+                'http://{}:{}/nodes/{}/pins/{}/'.format(self._host, self._port, self.pio.node, self.pio.pin),
```

Comment:
Since the URL for `set()` and `get()` is the same, maybe construct it in `on_activate()`?

Reply:
Done.